### PR TITLE
feat: support specify listen host via enviroment variable

### PR DESCRIPTION
**Pull request by ImSingee**
### Description

Support listen on specific host (other than default 127.0.0.1)

### Linked Issues

None

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -4,4 +4,5 @@ AI_API_KEY=
 # AI_MAX_OUTPUT_TOKENS=100
 AI_TEMPERATURE=0.5
 TRANSLATE_TYPE=ai # shortcut, ai
-# PORT=3000
\ No newline at end of file
+# PORT=3000
+# LISTEN=127.0.0.1
```
Comment (wibus-wee) [inline .env.example:8]: would it be better to use `HOST` as the key?

**Review by wibus-wee — CHANGES_REQUESTED** [review]
You may also need to go to `./src/types.ts` to modify `EnvConfig` and `KeyOfEnvConfig`. 

Thank you very much! ❤️

File: .env.example (hunk 1 above)
Comment (ImSingee) [reply .env.example:8]: Sure, use LISTEN is just my habit😂

**Comment by ImSingee** [comment]
@wibus-wee Hi, I have changed per your request

**Review by wibus-wee — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**